chipsec/hal/spi.py

```diff
@@ -853,3 +853,184 @@ def get_SPI_JEDEC_ID_decoded(self):
         part = JEDEC_ID.DEVICE.get( jedec_id, 'Unknown')
 
         return (jedec_id, manu, part)
+
+    def parse_spi_flash_descriptor(self, rom):
```

Comment:
I'm not sure about how this code has been refactored into the SPI HAL.  The code the processes the raw ROM data should be put in one or more library functions.  The register accesses should have interfaces added to the SPI HAL.  The spidesc util command should then be updated to call the desired functions and format the output.